third_party/xla_client/util.h

```diff
@@ -58,6 +58,16 @@ std::vector<typename C::value_type::element_type*> GetSharedPointers(
   return pointers;
 }
 
+template <typename T, typename C>
```

Comment:
This doesn't look particularly useful outside of XRT client, maybe leave it in `third_party/xla_client/xrt_computation_client.cc` until it gets more users. Also, can't we use `std::copy` instead?

Reply:
It's a utility function. If we bury it inside a cc file, we need to remember to look there as well when we wonder of something like that already exists.
Can't use std::copy there, as this marshal different types.

Reply:
I'm confused, `std::copy` works with different element types, what is the problem here?

Reply:
It does, but here (in the specific call site) we are narrowing from int64 to int.
It triggers narrowing warnings.
If we are OK with those, this API can turn to:

return std::vector<T>(a.begin(), a.end());

Reply:
Oh, heck, we have so many warnings already.
Dropped the API altogether.